src/application/utils/transaction.ts

```diff
@@ -274,16 +281,20 @@ export function createTaxiTxFromTopup(
   );
 
   const pset = Pset.fromBase64(taxiReply.topup.partial);
-  const initialNumberOfInputs = pset.inputs.length;
+  const initialNumberOfInputs = pset.globals.inputCount;
```

Comment:
whats the difference?

Reply:
no diff but it ensures we have the counter incremented by addInputs methods (if for any reason pset.inputs contains non-input object)